docs/resources/waf_dedicated_domain.md

```diff
@@ -177,30 +179,31 @@ The following arguments are supported:
   Only supports one traffic identifier.
   The [traffic_mark](#DedicatedDomain_traffic_mark) structure is documented below.
 
+<a name="DedicatedDomain_server"></a>
 The `server` block supports:
 
-* `client_protocol` - (Required, String, ForceNew) Protocol type of the client. The options include `HTTP` and `HTTPS`.
-  Changing this creates a new service.
+* `client_protocol` - (Required, String, ForceNew) Specifies the protocol type of the client. The options include `HTTP`
+  and `HTTPS`. Changing this creates a new service.
```

Comment:
The options include **HTTP** and **HTTPS**. Changing this creates a new service.

Reply:
OK